### ISSUE TITLE
remote url ssh login "permission denied"

### ISSUE THREAD

**Issue by weirders** [raveberry/raveberry#118]
Ive got a VPS up and running and made the raveberry user there.
created SSH keys on the pi, then ssh-copy-id them over to the server.
I can now login with ssh raveberry@host without pass.

When i start the remote function and run (systemctl status remote), i get the following error: 

```
remote.service - remote
   Loaded: loaded (/etc/systemd/system/remote.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Fri 2021-09-24 13:28:58 CEST; 11min ago
 Main PID: 9149 (code=exited, status=1/FAILURE)

Sep 24 13:28:57 raveberry systemd[1]: Started remote.
Sep 24 13:28:57 raveberry autossh[9152]: starting ssh (count 1)
Sep 24 13:28:57 raveberry autossh[9152]: ssh child pid is 9163
Sep 24 13:28:57 raveberry connect_to_remote[9149]: Permission denied, please try again.
Sep 24 13:28:57 raveberry connect_to_remote[9149]: Permission denied, please try again.
Sep 24 13:28:57 raveberry connect_to_remote[9149]: raveberry@host_ip_redacted: Permission denied (publickey,password).
Sep 24 13:28:57 raveberry autossh[9152]: ssh exited prematurely with status 255; autossh exiting
Sep 24 13:28:58 raveberry systemd[1]: remote.service: Main process exited, code=exited, status=1/FAILURE
Sep 24 13:28:58 raveberry systemd[1]: remote.service: Failed with result 'exit-code'.
```

Am i missing something glaringly obvious?

**Comment by raveberry** [comment]
Did you specify your remote information during the system install? If not, maybe the identity file is not at the expected location. You can try directly running the command from the script to see if it works:
`sudo /usr/bin/autossh -i /root/.ssh/raveberry_remote -N -R 127.0.0.1:<remote_port>:localhost:80 raveberry@<remote_ip>`

**Comment by weirders** [comment]
allrighty, ran the command, asked me for pass, indicating the SSH file wasnt valid.
I copied over the id_rsa from pi user to root it works now. 

I did do a system-install again after messing with the SSH keys, but the conf didnt change. Maybe it wasnt copied over because of that? Or should i have done the SSH keys as RPI root user?

**Comment by raveberry** [comment]
If /root/.ssh/raveberry_remote already exists, it is not overwritten. I did this because I like to be able to do the system-install from both my computer and the pi directly without changing the config, but the paths for the key differ. If it is not copied, the path does not cause problems.
I did not think that this will matter that much, but it is not the behavior you would expect. Maybe I will change this so the key is always overwritten.
The intended usage is that you have a keypair for your vps, created however you like. In the raveberry.yaml you pass the path to the key and the setup then copies it into /root/.ssh/raveberry_remote. If this does not work, I will have to look into it again.
